extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/openx.spec.js

```diff
@@ -0,0 +1,55 @@
+/*global describe, expect, it, jasmine, modules*/
+describe('ext.wikia.adEngine.lookup.prebid.adapters.openx', function () {
+	'use strict';
+
+	var mocks = {
+		instantGlobals: {
+			wgAdDriverOpenXPrebidBidderCountries: ['PL']
+		},
+		geo: {
+			isProperGeo: jasmine.createSpy('isProperGeo')
```

Comment:
Done https://github.com/Wikia/app/pull/12893/commits/87b95f4289a366921851ac0b5eb291f58cdd36c3
Why do you think it's better this way? For me it adds complexity: I need to define empty method, and the spy on that method instead of creating a spy directly.